packages/block-editor/src/components/contrast-checker/index.js

```diff
@@ -71,16 +83,19 @@ function ContrastChecker( {
 	const colordTextColor = colord( textColor || fallbackTextColor );
 	const hasTransparency =
 		colordBackgroundColor.alpha() !== 1 || colordTextColor.alpha() !== 1;
```

Comment:
Should we reduce the minimum alpha threshold here where `__experimentalEnableAlphaChecker === true`? Say to `0.8`, so the warning will only show when the transparency of either text or background falls below this?